package.json

```diff
@@ -3,29 +3,31 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "DEBUG=specmatic-order-backend-nodejs:* node ./src/bin/www",
-    "startWithDeps": "npm run startDepsAndWait && npm start",
-    "startDepsAndWait": "npm run startDeps & sleep 10",
-    "startDeps": "cross-env SPECMATIC_LOG_LEVEL=debug specmatic stub --port 9000 & specmatic kafka",
-    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules NODE_NO_WARNINGS=1 node ./node_modules/jest/bin/jest.js --collectCoverage --detectOpenHandles",
+    "start": "set DEBUG=specmatic-order-backend-nodejs:* && node ./src/bin/www",
+    "startWithDeps": "concurrently --kill-others \"npm run start\" \"specmatic stub --port 8080\" \"specmatic kafka --port 9000\"",
```

Comment:
Let's keep the ports as is. Keep the stub port as 9000 and we don't need to pass a port to kafka